showcase/app/components/page-components/button/code-fragments/with-loading-state.gts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+import Component from '@glimmer/component';
+import { on } from '@ember/modifier';
+import { tracked } from '@glimmer/tracking';
+
+import { HdsButton } from '@hashicorp/design-system-components/components';
+
+export interface CodeFragmentWithLoadingStateSignature {
+  Element: HTMLDivElement;
```

Comment:
[suggestion] technically is not needed, we're not applying the `...attributes` to any element and there's no `<div>` in the template.
```suggestion
```